.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -6,9 +6,9 @@
 
 set -e
 
-CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.version")
+CURRENT_QEMU_TAG=$(get_version "assets.hypervisor.qemu.tag")
```

Comment:
Hi~ @GabyCT So so so sorry for delaying the response. I got sidetracked by updating ARM CI kernel. [ci#169](https://github.com/kata-containers/ci/issues/169).
Here, we have specific qemu tag version for AArch64.
CURRENT_QEMU_TAG=$(get_version "assets.hypervisor.qemu.architecture.aarch64.tag")